src/main/java/com/pnu/ordermanagementapp/client/ClientDbAdapter.java

```diff
@@ -19,7 +20,7 @@ public ClientDbAdapter(ClientRepository repository) {
 
     @Override
     public List<Client> findAll() {
-        return (List<Client>) repository.findAll();
+        return ((List<Client>)repository.findAll()).stream().filter(i->i.isActive()).collect(Collectors.toList());
```

Comment:
Formatting, every method on new line.
Also` i->i.isActive()` might be replaced with method refference

Reply:
Ok